nifti2/nifti2_io.c

```diff
@@ -3398,7 +3399,7 @@ int64_t nifti_get_volsize(const nifti_image *nim)
 
     \return 1 on existence, 0 otherwise
 *//*--------------------------------------------------------------------*/
-int nifti_fileexists(const char* fname)
+static int nifti_fileexists(const char* fname)
```

Comment:
I guess we can see if anyone complains, but this removes the function from the library.  An alternate change might be to add the prototype to the .h file.

Reply:
So this illustrates why having no `-Wmissing-prototypes` warnings in the codebase is desirable. If this is was meant to be public API, we found a bug: namely that it was missing from the header file.

I can alter this change to put it in the header if you want.  I didn't know if it was meant to be public API or not.